atest/test/02_Content_Keywords/assertions.robot

```diff
@@ -23,7 +23,7 @@ Get Attribute Names Does Not Support Formatter:
     Get Attribute Names    id=two    ==    id
     [Teardown]    Formatter TearDown
 
-Get Attribute Names Does Not Support Formatter:
+Get Attribute Names Does Not Support Formatter2:
```

Comment:
Why `2` at the end?

Reply:
Because it had duplicate names. 
just cosmetics